src/components/UI/Cards/ProductCardWithPrice.vue

```diff
@@ -0,0 +1,29 @@
+<template>
+    <div :id="name | toId">
+        <div class="flex w-full h-full justify-center items-center">
+            <div class="flex flex-col w-2/5 h-auto justify-center items-center cursor-pointer">
+                <div style="background-image: url(https://images.unsplash.com/photo-1539185441755-769473a23570?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1951&q=80"
+                     class="flex bg-gray-300 h-32 md:h-56 w-24 md:w-4/5 mx-2 rounded-sm shadow-md bg-cover bg-top">
```

Comment:
here agine if you use specific width you don`t need to add all this width just make it full and you will get the right width

Reply:
Thanks for the feedback. Can you look at the current updates. I have updated the code for responsiveness. It would be kind of you if you could check them!